srcalc/src/main/webapp/WEB-INF/views/fragments/editVariableKey.jspf

```diff
@@ -0,0 +1,13 @@
+<%--
+A simple JSP fragment to edit a variable key. This content expects to be in a
+Spring <form> tag with the form object of type EditVariable and called
+'variable'.
```

Comment:
Yes, probably. Under ASRC-142 I'm creating a whole editVariablePage custom tag to avoid repetition so this include will be deleted anyway. Can I keep it how it is for now?